resources/components/ImageCard.vue

```diff
@@ -19,17 +19,37 @@
 				</ul>
 			</div>
 
-			<div class="wbmad-buttons">
-				BUTTONS GO HERE
+			<div class="wbmad-action-buttons">
+				<base-button
+					v-bind:classes="[ 'wbmad-action-buttons__publish' ]"
+					v-bind:callback="onPublish"
+					v-bind:primary="true"
+					v-bind:progressive="true"
+				>
+					<span v-i18n-html:machinevision-publish />
+				</base-button>
+				<base-button
+					v-bind:classes="[ 'wbmad-action-buttons__skip' ]"
+					v-bind:callback="onSkip"
+					v-bind:framed="false"
+				>
+					<span v-i18n-html:machinevision-skip />
+				</base-button>
 			</div>
 		</div>
 	</div>
 </template>
 
 <script>
+var Button = require( './base/Button.vue' );
+
 module.exports = {
 	name: 'ImageCard',
 
+	components: {
+		'base-button': Button
```

Comment:
We've discussed a prefix for base components (`mw` or `w` something) and this is another good reason for that (HTML elements that already use the name)